desertbot/message.py

```diff
@@ -1,7 +1,9 @@
 # -*- coding: utf-8 -*-
 from enum import Enum
+from typing import Dict
 import re
 
+from desertbot.desertbot import DesertBot
```

Comment:
And so is this, if I remove the other one. I assume more would show up if I continued.

I didn't know python 3's type hinting was implemented this badly =/